doc/languages-frameworks/python.section.md

```diff
@@ -20,6 +20,347 @@ ps
 : stands for nixpkgs python **p**ackage **s**et.
 
 
+### API {#python-API}
+
+`TODO`
+
+#### pkgs.fetchPypi {#python-pkgs-fetchPypi}
+
+`TODO`
+
+#### pkgs.mkShell.venvdir {#python-pkgs-mkShell-venvdir}
+
+`TODO`
+
+#### pkgs.${python} {#python-pkgs-xpythonx}
+
+`TODO`
+
+#### pkgs.${python}.buildEnv {#python-pkgs-xpythonx-buildEnv}
```

Comment:
@pennae Any ideas on how to represent something like `${var}` in the reference part? Going for especial characters makes the `nrd` break, and anyway it might be a good idea to not be too fancy with the character set in the links.